docs/docs/upgrading-node-js.md

```diff
@@ -0,0 +1,119 @@
+# Upgrading your Node.js version
+
+## Introduction
+
+## Gatsby's Node.js support policy
+
+Gatsby aims to support any version of Node that has a release status of _Current_, _Active_, or _Maintenance_. Once a major version of Node reaches _End of Life_ status Gatsby will stop supporting that version.
+
+Gatsby will stop supporting the _End of Life_ Node release in a minor version.
+
+Check [Node's releases document](https://github.com/nodejs/Release#nodejs-release-working-group) for version statuses.
+
+## What version of Node do I have?
+
+Run `node -v` in a terminal to see which version of Node you have.
+
+```
+node -v
+v10.16.0
+```
+
+This example shows Node version 10.
```

Comment:
Un-opinionated on this 🤷‍♂ could be worth doing. Up to you!

```suggestion
This example shows Node version 10, specifically v10.16.0.
```

Reply:
It's a good one 👍